polkadot/transaction-pool/src/lib.rs

```diff
@@ -59,6 +59,11 @@ use substrate_runtime_primitives::traits::{Bounded, Checkable, Hash as HashT, Bl
 pub use extrinsic_pool::txpool::{Options, Status, LightStatus, VerifiedTransaction as VerifiedTransactionOps};
 pub use error::{Error, ErrorKind, Result};
 
+/// Maximal size of a single encoded extrinsic.
+///
+/// See also substrate-consensus::MAX_TRANSACTIONS_SIZE
```

Comment:
Hm was unable to find this!

Reply:
Good catch, I meant `polkadot-consensus`.